crypto/crypto/src/multi_ed25519.rs

```diff
@@ -502,8 +505,16 @@ impl Signature for MultiEd25519Signature {
     ) -> Result<()> {
         // Public keys should be validated to be safe against small subgroup attacks, etc.
         precondition!(has_tag!(public_key, ValidatedPublicKeyTag));
-        let last_bit = bitmap_last_set_bit(self.bitmap);
-        if last_bit == None || last_bit.unwrap() as usize > public_key.length() {
+        let last_bit = match bitmap_last_set_bit(self.bitmap) {
+            Some(last_bit) => last_bit,
+            None => {
+                return Err(anyhow!(
+                    "{}",
+                    CryptoMaterialError::BitVecError("Failed to find last set bit".to_string())
+                ))
+            }
+        };
+        if last_bit as usize > public_key.length() {
```

Comment:
The rest of the changes look good, but we need to have a crypto expert review this as it is changing the expected invariants. cc @huitseeker

Reply:
it looks fine to me : )